### DISCUSSION TITLE
Turn off the safety feature of the laser scanner

### DISCUSSION THREAD

**Discussion by GoktugGurler95** [PilzDE/psen_scan_v2#325] [General] [answered]
Hello, when our vehicle enters a specific area, we want turn off the safety feature for a certain period of time.different zone configurations are not active in the structure in the XML configuration file. How do I turn it off temporarily?
Best Regards

**Comment by ct2034** [comment, marked as answer]
Thanks for your question. If I understand you correctly, you want to switch of the safety monitoring from ROS. You want to disable the feature that the Robot stops, when something intrudes the safety zone. Is this correct.
Unfortunately this is not possible. This would go against all safety regulation if ROS could just disbale the safety. You can switch the safety zones by physical inputs to the scanner. You can find more details in https://www.pilz.com/download/open/PSEN_sc_M_3_05_5_08-12_Op_Ma__1005055-EN-07.pdf at page 68.

**Comment by GoktugGurler95** [comment]
Because we need to turn off the PSen laser scanner, which we use in our autonomous mobile vehicle, in special cases. When taking charge and entering the autonomous charging station. The only case I understand from your answer and I also know is that I need to do muting by triggering the safety feature with IO pins.

**Reply by ct2034** [reply]
Exactly, that would be the only option
